azure-iot-provisioning-devicesdk/azure/iot/provisioning/devicesdk/transport/device_provisioning_constant.py

```diff
@@ -0,0 +1,23 @@
+# -------------------------------------------------------------------------
```

Comment:
I'd rename this file to just be `constant.py`.

Then, constants can be imported and used like:

```python
from transport import constant

timeout = constant.DEFAULT_TIMEOUT
```

The key is that when defining a module that will actually be accessed and used inline as a module, the name should be short.